### PR TITLE
Cleaned up command page/fixed dashboard

### PR DESCRIPTION
**Pull request by bcorwin**
Command page:
* Looks prettier (thoughts on that?)
image
* Commands still not working
 * It never seems to go the views.send_command
 * But it still redirects to views.commands
* Slow to load
 * It checks the values on each load
 * Maybe try to cache information?
 * Had to add sleeps between calls or most variables would not come back from the server

Dashboard:
* Needed to import timedelta to do.py
* Typo fixed

### CODE REVIEW

**Comment by bcorwin** [comment]
**"Final" commit. Good to merge**

File: monitor/views.py [hunk 1]
```diff
@@ -99,33 +100,31 @@ def api_in(request):
...
-        del request.session['command_status']
+    command = request.GET
+    
+    if command:
+        command_status = send2middleware(command.urlencode())
         error = command_status[0]
-        details = command_status[1]
+        details = command_status[1]        
 
     if not bool(error):
```
Comment (cutreth) [inline monitor/views.py]: Do we need these since you're initializing the variables as blank above?

File: monitor/views.py [hunk 2]
```diff
@@ -134,23 +133,43 @@ def commands(request):
...
-        if "on" in collection_status: collection_status = "On"
-        elif "on" in collection_status: collection_status = "Off"
-        else: collection_status = "?"
-        
+        if "on." in collection_status: collection_status = "on"
+        else: collection_status = "off"
+    print(collection_status)
+    
+    sleep(.1)
     s, logging_status = send2middleware("?code=L&dir=get")
```
Comment (cutreth) [inline monitor/views.py]: Can we break each of these into a separate function?

File: monitor/views.py [hunk 3]
```diff
@@ -134,23 +133,43 @@ def commands(request):
...
     if s != "Success": logging_status = "?"
     else:
-        if "on" in logging_status: logging_status = "On"
-        elif "on" in logging_status: logging_status = "Off"
-        else: logging_status = "?"
+        if "on." in logging_status: logging_status = "on"
+        else: logging_status = "off"
     
-    alert_status = "TBD"
+    sleep(.1)
```
Comment (cutreth) [inline monitor/views.py]: I also agree, let's use a session to cache these values (maybe start with a five minute expiration)? The trick will be making sure ad hoc updates update the sessions too. Question is if we track one instant and leave it be for ad hoc updates, or if we use something different for each one. I'm leaning towards the first option.

Comment (bcorwin) [inline monitor/views.py]: I'll leave it as is for now, but put a trello card in.

**Pull request merged**